packages/frontend-2/lib/workspaces/composables/management.ts

```diff
@@ -522,3 +598,71 @@ export const useSetDefaultWorkspaceRegion = () => {
     return res?.data?.workspaceMutations.setDefaultRegion
   }
 }
+
+export function useSsoAuth(workspaceSlug: string) {
+  const logger = useLogger()
+  const ssoError = ref<string | null>(null)
+
+  const { hasSsoEnabled, needsSsoLogin, error } = useWorkspaceSso({
+    workspaceSlug
+  })
+
+  if (error.value) {
```

Comment:
None of this is reactive. If useWorkspaceSso recalculates, these downstream values (ssoError) will remain out of date